gmd.py

```
@@ -322,6 +542,611 @@ def interactive_menu(env: CommandEnv) -> None:
+        ui.summary("Stats", ["No Markdown files found."])
+        return
+
+    try:
+        import frontmatter  # type: ignore
+    except Exception:  # pragma: no cover
+        frontmatter = None  # type: ignore
+
+    since_epoch = parse_input_time_to_epoch(getattr(args, "since", None))
+    until_epoch = parse_input_time_to_epoch(getattr(args, "until", None))
+
+    def _load_metadata(path: Path) -> Dict[str, Any]:
```

Ruff (0.15.9)

[error] Undefined name `Dict`

(F821)